### ISSUE TITLE
Newly added product

### ISSUE THREAD

**Issue by arijitmeesho** [RedisAI/vecsim-demo#11]
Let's say there are couple of new products which are to be indexed. Is there way to index them with all the indexed products? Or should we start indexing from the very begining?

**Comment by alonre24** [comment]
Of course, just send another `HSET` command with a hash that has the same mapping, and the value which corresponds to the vector field will be indexed as well. For example, by calling: 
`con.hset(new_key,mapping=new_item_metadata)`

**Comment by sameersharma00747** [comment]
I am getting this error when I am trying to create new index:
ResponseError: Index already exists

**Comment by DvirDukhan** [comment]
@sameersharma00747 
You don't need to re create the index. Just call additional `HSET` commands.

**Comment by sameersharma00747** [comment]
It worked. Thanks a lot. I just wanted to understand it in more detail. So after initial HNSW indexing on Redis whenever new products keep coming up we just have to pass HSET commands instead of reindexing from beginning?

**Comment by sameersharma00747** [comment]
What's the reason for getting zero as score while querying similar vector in Redis. All results returned are having 0 as score.
